src/lib/fileutils.ts

```diff
@@ -106,23 +99,56 @@ export const starterAzurePipelines = async (opts: {
         displayName: "Azure Login"
       },
       ...cleanedPaths.map(projectPath => {
+        const projectPathParts = projectPath
+          .split(path.sep)
+          .filter(p => p !== "");
+        // If a the projectPath contains more than 1 segment (a service in a
+        // mono-repo), use the last as the project name as it will the folder
+        // containing the Dockerfile. Otherwise, its a standard service and does
+        // not need a a project name
+        const projectName =
```

Comment:
Important bit: in a mono-repo, we append `-<service-folder-name>` to the image name so services can be differentiated in the ACR. In the case of a standard repo, just the `Build.DefinitionName` is used